File: logica/listaDeReforco/C/e4.c
#include <stdio.h>

int main(){
    char meses[30][12] = {"janeiro", "fevereiro", "março", "agosto", "maio", "junho", "julho", "agosto", "setembro", "outubro", "novembro", "dezembro"};
    int mes;
    printf("digite o numero do mes\n");
    scanf("%d", &mes);
    if( mes<0|| mes>12){
        printf("o numero e invalido\n");
        printf("digite o numero do mes");
        scanf("%d", &mes);
    }else{
        printf("o mes e %s", meses[mes-1]);
    }
}

File: logica/listaDeReforco/Python/e4.py
mes = int(input("digite o número do mês: "))
meses = ["janeiro", "fevereiro", "março", "agosto", "maio", "junho", "julho", "agosto", "setembro", "outubro", "novembro", "dezembro"]
if mes<0 or mes>12:
    print("número inválido")
    mes = int(input("digite o número do mês: "))
print(f"o mês é {meses[mes-1]}")
